authority/tls_test.go

```diff
@@ -418,7 +428,7 @@ ZYtQ9Ot36qc=
 			require.NoError(t, err)
 			testAuthority.db = &db.MockAuthDB{
 				MStoreCertificate: func(crt *x509.Certificate) error {
-					sassert.Equals(t, crt.Subject.CommonName, "smallstep test")
+					assert.Equal(t, crt.Subject.CommonName, "smallstep test")
```

Comment:
It's better to reverse the order of arguments for expected and actual, but not a blocker. I've been changing those whenever I saw them happening and felt like they didn't muddy the PRs I was working at the time.

Reply:
I know, but I've just find and replace everything with sassert 🙂

Fixed with 812ffd3